lms/static/js/spec/verify_student/make_payment_step_view_ab_testing_spec.js

```diff
@@ -0,0 +1,221 @@
+define([
+        'jquery',
+        'underscore',
+        'backbone',
+        'common/js/spec_helpers/ajax_helpers',
+        'common/js/spec_helpers/template_helpers',
+        'js/verify_student/views/make_payment_step_view'
+    ],
+    function( $, _, Backbone, AjaxHelpers, TemplateHelpers, MakePaymentStepView ) {
+        'use strict';
+
+        describe( 'edx.verify_student.MakePaymentStepView', function() {
+
+            var PAYMENT_PARAMS = {
+                orderId: "test-order",
+                signature: "abcd1234"
+            };
+
+            var STEP_DATA = {
+                minPrice: "12",
```

Comment:
It is comparing the values here 
https://github.com/edx/edx-platform/pull/10751/files#diff-e9fcdf2ef30e2bb0b208496847a0fe60R38
and val() method returns the value as string. One solution is to do some thing like this 
expect( parseInt(sel.val()) ).toEqual(price); then I can use minPrice:12 as integer.